packages/dd-trace/src/profiling/profiler.js

```diff
@@ -78,7 +83,6 @@ class Profiler extends EventEmitter {
   _capture (timeout) {
     if (!this._enabled) return
     this._lastStart = new Date()
-
     if (!this._timer || timeout !== this._config.flushInterval) {
```

Comment:
The `_flushInterval` will _never_ match the `_timeoutInterval` provided by `ServerlessProfiler`, so this loses the timer refresh optimization. Is there anything we can do to keep that? I won't block on this, but if there's a way to make it work that would be nice.

Reply:
Got it, I replaced it with the timeoutInterval I added so the timer refresh is used